--- popup.js	2017-11-15 14:57:38.000000000 -0600
+++ newone.js	2017-11-15 14:57:33.000000000 -0600
@@ -111,16 +111,18 @@
     // Closing popup.
     // Click the x event!
     jQuery('#popup-message-close').click(function () {
-      popup_message_disable_popup();
+      //popup_message_disable_popup();
     });
     // Click out event!
-    jQuery('#popup-message-background').click(function () {
-      popup_message_disable_popup();
+    jQuery('#popupacceptbutton').click(function () {
+      if($('#popupaccept').is(':checked') === true){
+        popup_message_disable_popup();
+      }      
     });
     // Press Escape event!
     jQuery(document).keypress(function (e) {
       if (e.keyCode === 27 && popupStatus === 1) {
-        popup_message_disable_popup();
+        //popup_message_disable_popup();
       }
     });
   }
